aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java

```diff
@@ -142,9 +137,6 @@ public SqlCommand createTableFor(@NonNull ModelSchema modelSchema) {
 
     /**
      * {@inheritDoc}
```

Comment:
You could probably just kill the entire comment and let the interface doco come through.